Lib/defcon/objects/glyph.py

```diff
@@ -722,19 +719,17 @@ def insertAnchor(self, index, anchor):
         This will post a *Glyph.Changed* notification.
         """
         try:
-            assert anchor.glyph != self
+            assert anchor.glyph is None
```

Comment:
Oh it passes on `AttributeError`, of course!
Sorry about that. I need a coffee ;)